packages/core/types/src/modules/documents/params/attributes/index.ts

```diff
@@ -82,14 +91,6 @@ export type GetValues<TSchemaUID extends UID.Schema> = {
   }
```

Comment:
Just so I understand, Schema attributes only contain attributes strictly listed in the schema.json file ? and not attributes such as the id or document id?

Reply:
Schema attributes are all the attributes present in a schema once loaded in Strapi, which doesn't include `id` or `documentId`